### ISSUE TITLE
preview of a comment

### ISSUE THREAD

**Issue by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#1677]
**Describe the new addon**
make a preview tab when writing a comment to see the preview of what the comment look like when posted like the github markdown thing,
#### example 
- show the preview of the emoji
- show clickable links

**Link to the new addon**
N/A

**Comment by RedGuy12** [comment]
This could also be done with other parts of profile, project, and studio pages, ie About Me, Notes and Credits, and studio Descriptions.
________________________________

**Comment by scratchusernamemrtbts** [comment]
any update?

**Comment by GrahamSH-LLK** [comment]
> any update?

No, there aren't any updates. This isn't a priority, but the fact that this is open means that we are open to it. The thing is, just because nobody responded doesn't mean that you can bug us to make it happen. Do _you_ want to code this? Thought not.

**Comment by Cool-Scratcher** [comment]
> > any update?
> 
> No, there aren't any updates. This isn't a priority, but the fact that this is open means that we are open to it. The thing is, just because nobody responded doesn't mean that you can bug us to make it happen. Do _you_ want to code this? Thought not.

Geez. Calm down.

**Comment by scratchusernamemrtbts** [comment]
I almost forgot about this

**Comment by Secret-chest** [comment]
Who would need this?

**Comment by return-misaPuding** [comment]
Yeah, the comment looks almost the same before posting and after posting, so I don't think there should be an addon for this
